<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchOptionsWidget</class>
 <widget class="QWidget" name="SearchOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>7</number>
     </property>
     <property name="verticalSpacing">
      <number>2</number>
     </property>
     <item row="3" column="1">
      <widget class="RXInput" name="pattern" native="true"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="select">
       <property name="text">
        <string>select</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="RXPathInput" name="filter" native="true"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Find</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="path"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Files</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="replaceLabel">
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QCheckBox" name="notBinary">
       <property name="text">
        <string>not binary</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
       <item>
        <widget class="RXReplaceInput" name="replacement" native="true"/>
       </item>
       <item>
        <widget class="QPushButton" name="preview">
         <property name="text">
          <string>preview</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="replace">
         <property name="text">
          <string>replace</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="fileCount">
       <property name="text">
        <string>fileCount</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Path</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="search">
       <property name="text">
        <string>search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RXPathInput</class>
   <extends>QWidget</extends>
   <header>widget/rxpathinput.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RXInput</class>
   <extends>QWidget</extends>
   <header>widget/rxinput.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RXReplaceInput</class>
   <extends>QWidget</extends>
   <header>widget/rxreplaceinput.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
